owa/app/js/components/data-visualization-configuration/DataVisualizationConfigurationBlock.tsx

```diff
@@ -26,7 +26,7 @@ const DataVisualizationConfigurationBlock = ({ reportConfig, reportData, reportI
   return (
     <div className="section">
       <ExpandableSection
-        headerComponent={<DataVisualizationConfigurationHeader reportConfig={reportConfig} />}
+        headerComponent={<DataVisualizationConfigurationHeader reportConfig={reportConfig} reportIdx={reportIdx}/>}
```

Comment:
You can add space before closing tag signature

Reply:
fixed